var/spack/repos/builtin/packages/r-circstats/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RCircstats(RPackage):
+    """Circular Statistics, from Topics in Circular Statistics"""
+
+    homepage = "https://cran.r-project.org/package=CircStats"
+    url      = "https://cran.r-project.org/src/contrib/CircStats_0.2-6.tar.gz"
```

Comment:
Needs `list_url`, should use `cloud` instead of `cran`. See https://spack.readthedocs.io/en/latest/build_systems/rpackage.html#list-url

Reply:
Can you use cloud for the homepage and url too?